src/base/_bootstrap-4.scss

```diff
@@ -43,7 +43,15 @@
 	}
 }
 
-/* Position relative */
+/* Positioning */
+.d-flex {
+	display: flex !important;
+}
```

Comment:
We will need to remove/update this because it was introduced by PR #9426

Reply:
Just wanted to clarify that #9426 added ``d-sm-flex``, which only activates flexbox layouts in small view and up.

This PR's ``d-flex`` class activates flexbox in all views.

So both classes target different scenarios and therefore might need to co-exist together.

Reply:
Agreed, but since all those CSS are in a single page, let wait until that page began to not be usable for web author and then let's reorganize it.